charts/llm-d-infra/templates/gateway-infrastructure/gatewayparameters.yaml

```diff
@@ -44,7 +60,6 @@ spec:
     podTemplate:
       extraLabels:
         gateway: custom
-    floatingUserId: true
```

Comment:
We need this to enable running on OCP, otherwise there is no way to make it work without significant scripting changes (unless something has changed that im not aware of)